ui/apps/platform/src/Containers/Collections/CollectionForm.tsx

```diff
@@ -268,44 +284,80 @@ function CollectionForm({
+                                        expressions (RE2 syntax).
+                                    </p>
+                                    <Divider className="pf-u-mb-lg" component="div" />
+                                    <RuleSelector
+                                        entityType="Deployment"
+                                        scopedResourceSelector={values.selectorRules.Deployment}
+                                        handleChange={onRuleSelectorChange}
+                                        validationErrors={errors.selectorRules?.Deployment}
+                                    />
+                                    <Label
+                                        variant="outline"
+                                        isCompact
+                                        className="pf-u-align-self-center"
+                                    >
+                                        in
+                                    </Label>
+                                    <RuleSelector
+                                        entityType="Namespace"
+                                        scopedResourceSelector={values.selectorRules.Namespace}
+                                        handleChange={onRuleSelectorChange}
+                                        validationErrors={errors.selectorRules?.Namespace}
+                                    />
+                                    <Label
+                                        variant="outline"
+                                        isCompact
+                                        className="pf-u-align-self-center"
+                                    >
+                                        in
+                                    </Label>
+                                    <RuleSelector
+                                        entityType="Cluster"
+                                        scopedResourceSelector={values.selectorRules.Cluster}
+                                        handleChange={onRuleSelectorChange}
+                                        validationErrors={errors.selectorRules?.Cluster}
+                                    />
+                                </Flex>
+
+                                <div className="pf-u-background-color-100 pf-u-p-lg">
                                     <Title headingLevel="h2">Attach existing collections</Title>
                                     <CollectionAttacher />
-                                </CardBody>
-                            </Card>
-                        </Flex>
-                        {action.type !== 'view' && (
-                            <div className="pf-u-p-lg pf-u-py-md">
-                                <>
+                                </div>
+                            </Flex>
+                            {action.type !== 'view' && (
+                                <div className="pf-u-p-lg pf-u-py-md">
```

Comment:
As a follow up, can you see if `ActionGroup` element meets the need with fewer `className` props?

https://www.patternfly.org/v4/components/form#actiongroup